.github/workflows/wdio-all-browsers.yml

```diff
@@ -0,0 +1,64 @@
+name: 'WDIO: All browsers'
+
+on:
+  workflow_dispatch:
+  workflow_call:
+    secrets:
+      JIL_SAUCE_LABS_USERNAME:
+        required: true
+      JIL_SAUCE_LABS_ACCESS_KEY:
+        required: true
+      JIL_NODE_NEW_RELIC_LICENSE_KEY:
+        required: true
+
+env:
+  BUILD_NUMBER: PR${{ github.event.number }}-job-${{ github.run_number }}-attempt-${{ github.run_attempt }}
+
+jobs:
+  chrome:
+    if: contains(fromJson('["workflow_dispatch"]'), github.event_name)
```

Comment:
None of the jobs will run on `workflow_call`

Reply:
Thanks for catching that